mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -82,6 +82,12 @@ examples:
     vars:
       repository_id: 'my-repository'
       description: 'example remote docker repository'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'artifact_registry_repository_cleanup'
```

Comment:
Need to add `min_version: beta` to the test

Reply:
Done, thanks.